backend/EasyMeets.Core/EasyMeets.Core.WebAPI/Controllers/AvailabilityController.cs

```diff
@@ -18,39 +17,45 @@ public AvailabilityController(IAvailabilityService availabilityService)
             _availabilityService = availabilityService;
         }
         
+        [Authorize]
```

Comment:
Please return attribute
And if you need to allow not registered user call endpoint you need to use allowanonymous attribute